piglow/piglow_test.go

```diff
@@ -0,0 +1,260 @@
+package piglow_test
```

Comment:
I agree with Rakyll, on top of the tests you wrote, we would like an example test that shows in the documentation and can be copied and pasted